src/main/java/tk/shanebee/hg/tasks/StartingTask.java

```diff
@@ -1,6 +1,10 @@
 package tk.shanebee.hg.tasks;
 
 import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
```

Comment:
these imports dont seem to be used/needed
```suggestion
import org.bukkit.entity.Player;
```